packages/test-utils/src/mocks/repair/v1/data.ts

```diff
@@ -0,0 +1,38 @@
+import faker from "faker/locale/en";
+
+export interface Repair {
```

Comment:
Maybe this should go to common

Reply:
We have a chicken/egg situation, I can do that once common is in. But common requires this.

Reply:
I see what you mean, Ideally we wouldn't release a package to get around a dependency issue.
It might have been easier to do a PR in Common with just the types etc. 
But I happy to get this through as is as long as we remember to remove it after.
I think typescript will be satisfied because the shapes will be the same but, if they start to differ it will be really hard to debug

Reply:
I'll put a PR in as soon as common is merged.